index.js

```diff
@@ -20,31 +20,37 @@ var typedArrays = {
 var slice = String.prototype.slice;
 var toStrTags = {};
 var gOPD = Object.getOwnPropertyDescriptor;
+var typedArray;
 if (hasToStringTag && gOPD && Object.getPrototypeOf) {
-	forEach(typedArrays, function (_, typedArray) {
-		var arr = new global[typedArray]();
-		if (!(Symbol.toStringTag in arr)) {
-			throw new EvalError('this engine has support for Symbol.toStringTag, but ' + typedArray + ' does not have the property! Please report this.');
+	/* jscs:disable disallowNodeTypes */
+	for (typedArray in typedArrays) {
```

Comment:
How is this an improvement over the `forEach` call? Loops should, in general, be avoided.